index.bs

```diff
@@ -7298,10 +7298,10 @@ for their contributions as our W3C Team Contacts.
   },
 
   "EduPersonObjectClassSpec": {
-    "publisher": ["Internet2 Middleware Architecture Committee for Education, Directory Working Group (MACE-Dir)"],
-    "title": "EduPerson Object Class Specification (200604a)",
-    "href": "http://software.internet2.edu/eduperson/",
-    "date": "March 15, 2016"
+    "publisher": "Research and Education FEDerations group - REFEDS.org",
+    "title": "EduPerson",
+    "href": "https://refeds.org/eduperson",
+    "date": "on-going"
```

Comment:
Change "on-going" to "ongoing", since it's a single word.  Then I'm good with this.

Reply:
done in 04919d7